source/language/subroutines.rst

```diff
@@ -14,6 +14,10 @@ the subroutine by reference or name, while classical types are passed in by valu
 All arguments are declared together with their type. For example, ``qubit ancilla``
 defines a quantum bit argument named ``ancilla``.
 
+A given qubit can passed at most once in any subroutine call.  Different
```

Comment:
```suggestion
A given qubit can be passed at most once in any subroutine call.  Different
```

Reply:
Thanks - this is what happens when I try to make two PRs in 3 minutes!